src/connectionhealthpolicy/BaseConnectionHealthPolicy.ts

```diff
@@ -10,12 +10,22 @@ export default class BaseConnectionHealthPolicy implements ConnectionHealthPolic
   protected minHealth: number;
   protected maxHealth: number;
   protected currentHealth: number;
+  readonly canonicalNameForLogging?: string;
```

Comment:
Anything wrong with just calling this `name`? Can't imagine we would have another name member in the future.

Reply:
Curse of overthinking :sigh:. will change